### PR TITLE
Added more functionality (task 3)

### PR DESCRIPTION
**Pull request by melsem08**

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      .then((response) => {
+        const categories = response.body.categories;
+        expect(categories.length).toBe(4);
+        categories.forEach((category) => {
+          expect(typeof category.slug).toBe("string");
+          expect(typeof category.description).toBe("string");
+        });
+      });
+  });
+  test.only("GET - status 404 - responds with error when passed unavailable route", () => {
```
Comment (MKD01) [inline __tests__/endpoints.test.js:34]: Please dont forget to remove .only's before pushing

**Review by MKD01 — CHANGES_REQUESTED** [review]
Please remove the .only on your test and resubmit this pr

File: app.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories.controller");
+
+const app = express();
+app.use(express.json());
+// console.log("Hi, I'm in app");
```
Comment (Keb84) [inline app.js:6]: Please remove your app.use(express.json()) at this point we you are only making a GET request and so this line is unnecessary 👀

File: controllers/categories.controller.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const { selectCategories } = require("../models/categories.model");
+
+exports.getCategories = (request, response) => {
+  //   console.log("Hi, I'm in controller");
+  selectCategories().then((categories) => {
+    response.status(200).send({ categories: categories });
+  });
+};
```
Comment (Keb84) [inline controllers/categories.controller.js:8]: Please remove all your console.logs. These shouldn't be in your code when your commit and push 🙂

**Review by Keb84 — CHANGES_REQUESTED** [review]
Really nice so far Semen. Please make the amendments and resubmit 🙂

**Review by leekli — APPROVED** [review]
Thanks Semen, approved :) 

You are free to merge this branch, if you haven't merged yet then please do follow the nc notes on this: https://notes.northcoders.com/courses/js-back-end/kanban-and-git-branches

**Pull request merged**